fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/SparkDpp.java

```diff
@@ -640,6 +642,22 @@ private StructType createScrSchema(List<String> srcColumns) {
         return srcSchema;
     }
 
+    // This method is to keep the splitting consistent with broker load / mini load
+    private String[] splitLine(String line, char sep) {
```

Comment:
if `line` is an empty string, this method should return an empty string array.
But here you will return a string array with one empty string in it.

Reply:
fixed it